pkg/storage/config_test.go

```diff
@@ -0,0 +1,40 @@
+package storage
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestName(t *testing.T) {
+	postgresConfig := PostgresConfig{"localhost", "99", "me!", "mypassword!", "name"}
+	assert.Equal(t, "Postgres", postgresConfig.Name())
+}
+
+func TestShouldReturnSame(t *testing.T) {
+	postgresConfigComponent := PostgresConfigComponent{}
+	postgresConfig := postgresConfigComponent.Settings()
+	assert.NotNil(t, postgresConfig)
+	assert.Empty(t, postgresConfig.DatabaseName)
+}
+
+func TestShouldMakeNewDB(t *testing.T) {
+	called := false
+	postgresConfig := PostgresConfig{}
+	originalDBInitFn := dbInitFn
```

Comment:
Doing this creates a constraint whereby we can never run tests in parallel because this relies on global state. Why aren't we able to use the sql mock here that is used elsewhere in the test instead?

Reply:
sqlmock does not offer a mock for `Open`.  I could, though, put a reference to the function in the struct rather than making it a global.  Yeah?